kvmagent/kvmagent/plugins/nvram/nvram_local.py

```
@@ -0,0 +1,173 @@
+
+        if is_mounted:
+            ret = bash.bash_r("umount %s" % shlex.quote(mount_path))
+            if ret != 0:
+                bash.bash_r("umount -l %s" % shlex.quote(mount_path))
+
+    def _detach_loop_device_by_file(self, nvram_install_path):
+        loop_dev = self._find_loop_block_device_by_file(nvram_install_path)
+        
+        if loop_dev:
+            if os.path.exists(loop_dev):
+                r, o, e = bash.bash_r("losetup -d %s" % shlex.quote(loop_dev))
```

Ruff (0.15.2)

[warning] Unpacked variable `o` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)